backends/ebpf/tests/p4testdata/counters.p4

```diff
@@ -56,11 +56,16 @@ control ingress(inout headers hdr,
         send_to_port(ostd, egress_port);
     }
 
+    action do_forward_2(PortId_t egress_port) {
+        action_cnt.count((bit<32>)hdr.ethernet.etherType);
+        send_to_port(ostd, egress_port);
+    }
+
     table tbl_fwd {
         key = {
             istd.ingress_port : exact;
         }
-        actions = { do_forward; NoAction; }
+        actions = { do_forward; do_forward_2; NoAction; }
```

Comment:
why do we need the second `do_forward` ?

Reply:
Just another way to call a Counter method in an action - with action argument and with header field. I used this to check if this also compiles correctly.